Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -193,19 +193,39 @@ private string GetFixedTableHeaderMaxHeight()
             return FixedHeaderDataGridMaxHeight;
     }
 
+    /// <summary>
+    /// Returns a list of all the columns that are currently associated with this datagrid.
+    /// </summary>
+    /// <returns>A read-only list of all columns</returns>
+    public IReadOnlyList<DataGridColumn<TItem>> GetColumns() => Columns.AsReadOnly();

```

Comment:
Should we just make the existing Columns API public instead of adding a new `GetColumns()` API?

Reply:
I don't mind either way. 
But I prefer this approach because:
-  This just conveys a better meaning that they shouldn't really be mutated (altought we know it's possible)
- Makes it harder to mutate it (again we know it's unfortunately possible anyway)

Reply:
Please resolve this if you're ok with this.